website/docs/r/orbital_contact.html.markdown

```diff
@@ -0,0 +1,143 @@
+  title_line        = "AQUA"
+
+  tags = {
+    aks-managed-cluster-name = "9a57225d-a405-4d40-aa46-f13d2342abef"
+  }
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "example-vnet"
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_subnet" "example" {
+  name                 = "example-subnet"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.1.0/24"]
+
+  delegation {
+    name = "orbitalgateway"
+
+    service_delegation {
+      name = "Microsoft.Orbital/orbitalGateways"
+      actions = [
+        "Microsoft.Network/publicIPAddresses/join/action",
+        "Microsoft.Network/virtualNetworks/subnets/join/action",
+        "Microsoft.Network/virtualNetworks/read",
+        "Microsoft.Network/publicIPAddresses/read",
+      ]
+    }
+  }
+}
+
+resource "azurerm_orbital_contact_profile" "example" {
+  name                              = "example-contactprofile"
+  resource_group_name               = azurerm_resource_group.example.name
+  location                          = azurerm_resource_group.example.location
+  minimum_variable_contact_duration = "PT1M"
+  auto_tracking                     = "disabled"
+  links {
+    channels {
+      name                 = "channelname"
+      bandwidth_mhz        = 100
+      center_frequency_mhz = 101
+      end_point {
+        end_point_name = "AQUA_command"
+        ip_address     = "10.0.1.0"
+        port           = "49153"
+        protocol       = "TCP"
+      }
+    }
+    direction    = "Uplink"
+    name         = "RHCP_UL"
+    polarization = "RHCP"
+  }
+  network_configuration_subnet_id = azurerm_subnet.example.id
+}
+
+resource "azurerm_orbital_contact" "example" {
+  name                   = "example-contact"
+  spacecraft_id          = azurerm_orbital_spacecraft.example.id
+  reservation_start_time = "2020-07-16T20:35:00.00Z"
+  reservation_end_time   = "2020-07-16T20:55:00.00Z"
+  ground_station_name    = "WESTUS2_0"
+  contact_profile_id     = azurerm_orbital_contact_profile.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Contact. Changing this forces a new resource to be created.
+
+* `spacecraft_id` - (Required) The ID of the spacecraft which the contact will be made to.
+
+* `reservation_start_time` - (Required) Reservation start time of the Contact.
+
+* `reservation_end_time` - (Required) Reservation end time of the Contact.
+
+* `ground_station_name` - (Required) Name of the Azure ground station.
+
+* `contact_profile_id` - (Required) ID of the orbital contact profile.
```

Comment:
Could we add "Changing this forces a new resource to be created." to these?

Reply:
Will update.